src/components/sections/splitSection/splitSection.module.css

```diff
@@ -14,3 +13,14 @@
     grid-template-columns: 1fr;
   }
 }
+
+/* TODO: This is just to get some gap between the title and the section above in the jobs page.
+In the calculator page, the title is not there, 
+and will not create a gap. A better solution is to go through 
+how section and its components are structured. Now each 
+component has a margin/padding, but it would be better to have 
+a spesific gap in the parent
+*/
+.title {
+  margin-top: 3rem;
+}
```

Comment:
Noe som hadde vært tøft hadde vært å ha en global "section"-klasse som hadde alternativer for bredde og ellers hadde samme gap og styling på alle, og som vi kunne puttet på alle sections. Eller noe lignende. Da hadde det vært neat.